Courir/Courir/GameNetworkPortal.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  GameNetworkPortal.swift
+//  Courir
+//
+//  Created by Hieu Giang on 20/3/16.
+//  Copyright © 2016 NUS CS3217. All rights reserved.
+//
+
+import Coulomb
+import MultipeerConnectivity
+
+internal protocol GameNetworkPortalConnectionDelegate {
```

Comment:
It's to explicitly say that this protocol is only available within the project. One style guide says it should be left out, another says it should be there for top-level declaration. How are you guys doing it? I guess we can be consistent about it.